Gemfile

```diff
@@ -5,7 +5,7 @@ source 'https://rubygems.org'
 group :release do
   gem 'emeril'
   gem 'rake'
-  gem 'chef', '~> 12.22'
+  gem 'chef', '= 14.10.9'
```

Comment:
The CI tests with chef 13.4, so maybe we should pin that version. Also, is there any reason to pin an exact minor and not use `~>`>

Also, note that emeryl is not compatible with chef > 12. But the right way to solve this is to get rid of emeryl and find a different way to do the releases... The current way is broken in so many ways anyway.

Reply:
With the version `12.22`, the rspec tests do not run and some tests are failing because of default values management (empty array in `12.22`, nil in most recent versions).
I've chosen the same version as the CI:
```bash
#!/bin/bash -eo pipefail
chef --version
Chef Development Kit Version: 3.8.14
chef-client version: 14.10.9
delivery version: master (9d07501a3b347cc687c902319d23dc32dd5fa621)
berks version: 7.0.7
kitchen version: 1.24.0
inspec version: 3.6.6
```

Reply:
👍 Checking again, it looks like we use 13 in Travis and 14 in Circle. When we unify the CIs we should make sure to test both versions (and probably 15 as well).

Reply:
And yup, tests don't pass in 12. The reason 12 was pinned here was because of Emeryl.

Reply:
Yup, I checked only on CircleCI. I get it now. Thank you!